endowed-positions.md

```diff
@@ -0,0 +1,21 @@
+# Endowed Positions
+
+Endowed Positions is a subsection of the Stanford Giving site. 
+
+The main route `/endowed-positions` is a page built via Storyblok. The sub-routes are built and controlled via Gatsby. Each school, center, institute, or program has its own sub-route i.e., `/endowed-positions/graduate-school-of-business`. Schools/centers are added or removed via [ENDOWED_POSITIONS_MAP.json](https://github.com/SU-SWS/ood_giving_site/blob/dev/src/constants/ENDOWED_POSITIONS_MAP.json).
+
+Search results are another sub-route i.e., `/endowed-positions/search?term=james`. This route will provide list of results of which each one is clickable. When one of the results is clicked a query param of the results' index is appended and the UI is changed to show only that result. There were potential plans to expand this but this functionality matched that of the previous Endowed Positions site. Search is powered by [Fuse.js](https://www.fusejs.io/).
+
+## Endowed Positions Files
+
+- [React component files](https://github.com/SU-SWS/ood_giving_site/tree/dev/src/components/endowed-positions)
+- [Dataset JSON](https://github.com/SU-SWS/ood_giving_site/tree/dev/src/fixtures)
+- [School/Center Map JSON](https://github.com/SU-SWS/ood_giving_site/tree/dev/src/fixtures)
```

Comment:
@wmui51 Is the link to School/Center Map JSON be this instead?
https://github.com/SU-SWS/ood_giving_site/blob/dev/src/constants/ENDOWED_POSITIONS_MAP.json